polygon-id/issuer/.env-api

```diff
@@ -7,7 +7,8 @@ ISSUER_API_UI_ISSUER_LOGO=
 ISSUER_API_UI_SCHEMA_CACHE=false
 ISSUER_API_IDENTITY_METHOD=polygonid
 ISSUER_API_IDENTITY_BLOCKCHAIN=polygon
-ISSUER_API_IDENTITY_NETWORK=mumbai
+ISSUER_API_IDENTITY_NETWORK=amoy
```

Comment:
also, update readme with the correct instructions

Reply:
fxied